app.js

```diff
@@ -8,7 +8,15 @@ const {
 const connector = new LCUConnector();
 const { app, dialog } = electron;
 const { BrowserWindow } = electron;
+const DiscordRPC = require('discord-rpc');
```

Comment:
Imports should be before definitions.
1. Absolute imports
2. Relative imports
3. Variables/Constants definitions

Reply:
I'll add an eslint file this weekend just to make it clearer for future PRs.